container/typings/LuigiCompoundContainer.svelte.d.ts

```diff
@@ -1,7 +1,3 @@
-export declare interface UserSettings {
-  [key: string]: number | string | boolean;
-}
-
```

Comment:
removed since it doesn't seem to be used anywhere else but here, and it can be represented by an 'Object' too as well. Same for other interfaces in ...Container.d.ts